APMrover2/sailboat.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "Rover.h"
```

Comment:
This isn't really needed here, only AP_Param.h is needed.

We should probably add GPL license header here.

Reply:
added the header, tuns out we didn't actually need to include anything here.